README.md

```diff
@@ -1,2 +1,105 @@
 # BETYdb-YABA
 Yet Another BETYdb API (for metadata upload)
+
+
+## Setup BETYdb
+
+Following commands can be used to initialize the database to be used 
+with YABA development.
+
+```bash
+docker-compose up -d postgres
+docker-compose run --rm bety initialize
+docker-compose run --rm bety sync
+```
+
+
+## Running the App
+
+### Bring up container for YABA-API
+
+```sh
+docker-compose up -d yaba_api
+```
+
+## Running the App (without initialization and synchronization of bety)
+
+### Bring up the containers at once
+
+#### Once initialization and synchronization of bety database is done,the below command can be directly used to start the app.(Skip the "Setup BETYdb" step) 
+```sh
+# Bring up full stack
+docker-compose up
+```
+
+
+## How to hit the endpoints
+
+Following endpoints can be used to upload Metadata to respective tables in bety.
+
+Experiments: (to experiments table):
+
+```sh
+curl -F "fileName=@input_files/experiments.csv"   \
+     http://localhost:5001/yaba/v1/experiments?username=guestuser
```

Comment:
This is returning code 402 and no other information.... should say something like "successfully uploaded and $n records to $table table"